OrbitGl/TriangleToggle.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TRIANGLE_TOGGLE_H_
+#define ORBIT_GL_TRIANGLE_TOGGLE_H_
+
+#include "PickingManager.h"
+
+#include <functional>
+
+class TimeGraph;
+class GlCanvas;
+
+class TriangleToggle : public Pickable {
+ public:
+  using StateChangeHandler = std::function<void(bool)>;
+  explicit TriangleToggle(bool initial_state, StateChangeHandler handler,
```

Comment:
Another case where having a boolean as a function argument (in this case public API), is unclear. Reading this, I don't know what the two possible initial states mean. Is the toggle collapsed or not? Would also suggest to use an enum here.

Reply:
Done.